app/models/concerns/orchestration/compute.rb

```diff
@@ -212,8 +213,9 @@ def delComputeUpdate
 
   def compute_update_required?
     return false unless compute_resource.supports_update? && !compute_attributes.nil?
-    old.compute_attributes = compute_resource.vm_compute_attributes_for(uuid)
-    compute_resource.update_required?(old.compute_attributes, compute_attributes)
+    attrs = compute_resource.vm_compute_attributes_for(uuid)
+    old.compute_attributes = attrs if old
```

Comment:
@houndci-bot: Again, this is only possible with Ruby >= Ruby 2.3.0. If I recall correctly the Foreman project supports older rubies.